src/pymor/core/defaults.py

```diff
@@ -382,12 +388,41 @@ def set_defaults(defaults):
         Dictionary of default values. Keys are the full paths of the default
         values (see :func:`defaults`).
     """
+    from pymor.tools import mpi
+    if mpi._event_loop_running and mpi.rank0:
+        mpi.call(_set_defaults, defaults)
     try:
         _default_container.update(defaults, type='user')
     except KeyError as e:
         raise KeyError(f'Error setting defaults. Key {e} does not correspond to a default')
 
 
+def get_defaults(user=True, file=True, code=True):
+    """Get |default| values.
+
+    Returns all |default| values as a dict. The parameters can be set to filter by type.
+
+    Parameters
+    ----------
+    user
+        If `True`, returned dict contains defaults that have been set by the user, usually with :func:`set_defaults`.
```

Comment:
```suggestion
        If `True`, returned dict contains defaults that have been set by the user
        with :func:`set_defaults`.
```

`set_defaults` should be the only public api to change a default.

Reply:
Thanks for your reviews! Could you please add that this pr fixes #1329 or close the issue afterwards?